src/citrine/resources/predictor_evaluation_workflow.py

```diff
@@ -1,83 +1,21 @@
 """Resources that represent both individual and collections of workflow executions."""
-from typing import Optional, List
 from uuid import UUID
 
 from citrine._rest.collection import Collection
-from citrine._rest.resource import Resource
-from citrine._serialization import properties
 from citrine._session import Session
-from citrine.informatics.predictor_evaluator import PredictorEvaluator
-
-
-class PredictorEvaluationWorkflow(Resource['PredictorEvaluationWorkflow']):
-    """[ALPHA] A process for evaluating a predictor
-
-    Parameters
-    ----------
-    uid: str
-        Unique identifier of the predictor evaluation workflow
-    name: str
-        name of the predictor evaluation workflow
-    description: str
-        the description of the predictor evaluation workflow
-    evaluators: List[PredictorEvaluator]
-        the list of evaluators to apply to the predictor
-
-    """
-
-    _response_key = 'WorkflowExecutions'
```

Comment:
@maxhutch Was this a copy-paste error or intentional?

Reply:
Copy-paste.  I didn't change much plumbing.